client/src/store/modules/traffic.js

```diff
@@ -0,0 +1,38 @@
+import axios from '@/plugins/axios';
+
+const namespaced = true;
+
+const state = () => ({
+    entries: []
+});
+
+const getters = {
+    total: state => state.entries.length,
+    perPage: state => page => state.entries.filter(entry => entry.page === page).length,
+    perSource: state => source => state.entries.filter(entry => entry.source === source).length
+};
+
+const mutations = {
+    SET_ENTRIES: (state, entries) => (state.entries = entries)
+};
+
+const actions = {
+    async fetchEtriesByTime({ commit }, between) {
+        const params = {
```

Comment:
const params = { filter : { between } };

Reply:
@daniel-pupczyk-rmtcfm-com Ok. Gonna change it.

Reply:
Yeah, we are sending filter as object, it's Simple REST API standard:
Here you will find examples:
https://marmelab.com/react-admin/DataProviders.html

ie.
`GET http://my.api.url/posts?sort=["title","ASC"]&range=[0, 24]&filter={"title":"bar"}`

Reply:
@rafal-ksiazek-rmtcfm-com Not as object, but as JSON and it should be parsed in code. But if I do `http://my.api.url/posts?filter[title]=bar&filter[author]=Jhon Doe&filter[info][buzz]=abc`, express will auto parse it to object `filter: { title: 'bar', author: 'Jhon Doe', info: { buzz: 'abc' } }`. Anyway, I changed it.